src/login/AzureLoginHelper.ts

```diff
@@ -411,11 +411,15 @@ export class AzureLoginHelper {
 		});
 	}
 
-	private async noSubscriptionsFound(): Promise<void> {
-		const open: MessageItem = { title: localize('azure-account.open', "Open") };
-		const response: MessageItem | undefined = await window.showInformationMessage(localize('azure-account.noSubscriptionsFound', "No subscriptions were found. Set up your account at https://azure.microsoft.com/en-us/free/."), open);
-		if (response === open) {
-			void openUri('https://azure.microsoft.com/en-us/free/?utm_source=campaign&utm_campaign=vscode-azure-account&mktingSource=vscode-azure-account');
+	private async showNoSubscriptionsFoundNotification(): Promise<void> {
+		const noSubscriptionsFound = localize('azure-account.noSubscriptionsFound', 'No subscriptions were found. Setup your account if you have yet to do so or check out our troubleshooting page for common solutions to this problem.');
+		const setupAccount = localize('azure-account.setupAccount', 'Setup Account');
+		const openTroubleshooting = localize('azure-account.openTroubleshooting', 'Open Troubleshooting');
+		const response = await window.showInformationMessage(noSubscriptionsFound, setupAccount, openTroubleshooting);
+		if (response === setupAccount) {
```

Comment:
I think adding telemetry to this (if possible, not sure if the telemetryReporter is set up yet) would be useful.  It'd be a nice way to tell how many people are running into errors because they just don't have accounts or have an account that isn't showing up.